fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -410,28 +410,26 @@ public void dropTable(String tableName) {
         return views;
     }
 
-    public List<Table> getTablesOnIdOrderOrThrowException(List<Long> tableIdList) throws MetaNotFoundException {
-        List<Table> tableList = Lists.newArrayList();
+    public List<Table> getTablesOnIdOrderNullable(List<Long> tableIdList) {
```

Comment:
Add comment to explain behavior of this method.
And it is strange to just ignore missing table instead of throwing exception.
The method name can not reflect this behavior, may leading misuse